### ISSUE TITLE
Upcoming page looks broken after .legacy-page margin changes

### ISSUE THREAD

**Issue by zarino** [mysociety/theyworkforyou#1009]
#1003 limited line lengths on static pages, including `.legacy-page` pages.

Unknown to me, this also affected the "Upcoming" page, which isn't ideal:

![screen shot 2015-10-28 at 13 25 28](https://cloud.githubusercontent.com/assets/739624/10789820/935a299e-7d77-11e5-9da8-8c24c2f00a10.png)

Really this page should use the same sections-and-sidebars styling as the other business pages.

**Comment by dracos** [comment]
This also affects http://www.theyworkforyou.com/api/ looks like.

**Comment by zarino** [comment]
@dracos – I’ve started this in the branch [1009-todays-business-page](https://github.com/mysociety/theyworkforyou/compare/1009-todays-business-page), but because I've switched to using the new templating system (as we have with other pages, recently) there are a few things that don't work:

* The main page navigation is missing
* I can't work out how to get the future calendars displaying in the sidebar
* An exception gets raised from `/data/twfy/classes/Utility/Calendar.php` when you try to display a date with no events (because it still expects the `$PAGE` class to be there).

The core of the change is using the `business-section` styles, inside of a `full-page__row` wrapper, as we do elsewhere on the site. We'll want to do the same for the API pages, but there are a lot of them, so before I started, I wanted to check I'm even doing the right thing.
